### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Aimsucks**
- Completely overhauled README.md
- Added screenshots for auto-leveling interface and dungeon path
- Explained main functionality of plugin
- Added table for features before, during, after, and between runs
- Added a license disclosure
- Added _a lot_ of links to Discords

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,138 +1,70 @@
...
+
+Creating paths is also supported and easily accomplished within AutoDuty by using the `Build` tab at the top. For path status, please refer to the [pinned spreadsheet](https://discord.com/channels/1001823907193552978/1236757595738476725/1243059104528994334) in the AutoDuty channel of the [support Discord](https://discord.gg/punishxiv).
+
+# Automatic Leveling
+
+<img align="right" height="250" src="Assets/leveling.png">
+
+AutoDuty has a leveling feature that will continuously run you through the "correct" dungeon to level through. The dungeon is typically determined by path quality, support for boss fights, and obviously the level of the dungeon.
+
+With AutoDuty, you can level a job from 15 to the max level very quickly and fully AFK, provided you have the correct gear and the option in the AD configuration to automatically the best gear.
```
Comment (adambennett55) [inline README.md:27]: equip?

File: README.md [hunk 2]
```diff
@@ -1,138 +1,70 @@
...
+- [Rotation Solver Reborn](https://raw.githubusercontent.com/FFXIV-CombatReborn/CombatRebornRepo/main/pluginmaster.json): Automatic rotation execution, all jobs supported
+- [Veyn's Boss Mod](https://puni.sh/api/repository/veyn) or [BossmodReborn](https://raw.githubusercontent.com/FFXIV-CombatReborn/CombatRebornRepo/main/pluginmaster.json): Automatic fight execution for bosses
+
+## Optional Plugins
+
+The following plugins are optional. They integrate with AutoDuty well and in some cases can be triggered by AD itself.
+
+- [Deliveroo](https://plugins.carvel.li): Automatically turns in items to your Grand Company
+- [AutoRetainer](https://love.puni.sh/ment.json): Automatically manages your retainers and submersibles as well as sends them out on ventures
+
```
Comment (adambennett55) [inline README.md:61]: can you add gearsetter here

File: README.md [hunk 3]
```diff
@@ -1,138 +1,70 @@
...
+# Getting help
+
+When you've found a bug or think you have some issue with the plugin, please ask in [Discord](https://discord.gg/punishxiv). It might be a known issue or people might be able to help you quickly. Additionally, sometimes it might be an issue with one of the required plugins. Users in Discord will be able to help triage the issue and send you to the right place.
+
+Best practice is to not say "I died in this dungeon and I don't know why." Please make sure to include as much detail as possible. What boss were you on? Did you get stuck in a specific spot?
+
+For support with Veyn's Boss Mod and vnavmesh, please ask in the [Puni.sh Discord server](https://discord.gg/punishxiv). For support with BossModReborn and Rotation Solver Reborn, please ask in the [Combat Reborn Discord server](https://discord.gg/p54TZMPnC9).
```
Comment (adambennett55) [inline README.md:68]: these discord links i would prefer direct to channel links:
ad -> https://discord.com/channels/1001823907193552978/1236757595738476725
vbm -> https://discord.com/channels/1001823907193552978/1191076246860349450

File: README.md [hunk 4]
```diff
@@ -1,138 +1,70 @@
...
-[stars-url]: https://github.com/ffxivcode/AutoDuty/stargazers
-[issues-shield]: https://img.shields.io/github/issues/ffxivcode/AutoDuty.svg?style=for-the-badge
-[issues-url]: https://github.com/ffxivcode/AutoDuty/issues
+# AutoDuty
+
+<img align="right" width="150" height="150" src="logo.png">
+
+AutoDuty (AD) is a Dalamud plugin for FFXIV that serves as a tool to assist in the creation and following of paths through dungeons and duties. It's designed to enable automatic looping of instanced content with Duty Support, Trusts, or Squadrons.
+
+Support for it can be found in the [Puni.sh Discord server](https://discord.gg/punishxiv).
```
Comment (adambennett55) [inline README.md:7]: these discord links i would prefer direct to channel links:
ad -> https://discord.com/channels/1001823907193552978/1236757595738476725

File: README.md [hunk 5]
```diff
@@ -1,138 +1,70 @@
...
+
+# Dungeon Paths
+
+<img align="right" height="250" src="Assets/paths.png">
+
+Each dungeon supported by AutoDuty has a pre-configured path laid out, which should complete the dungeon without any issues. If any issues are encountered, make sure to reach out in the [support Discord](https://discord.gg/punishxiv).
+
+The paths will also run over all loot containers and, when configured, will pick up the loot and do something with it while either in between dungeon runs or after your set amount of runs are over.
+
+Creating paths is also supported and easily accomplished within AutoDuty by using the `Build` tab at the top. For path status, please refer to the [pinned spreadsheet](https://discord.com/channels/1001823907193552978/1236757595738476725/1243059104528994334) in the AutoDuty channel of the [support Discord](https://discord.gg/punishxiv).
```
Comment (adambennett55) [inline README.md:19]: these discord links i would prefer direct to channel links:
ad -> https://discord.com/channels/1001823907193552978/1236757595738476725

**Review by adambennett55 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Aimsucks) [reply README.md]: Fixed in [2e3e495](https://github.com/ffxivcode/AutoDuty/pull/317/commits/2e3e49574234e05830d2c685c7d4677bdcb37b43)

File: README.md (hunk 2 above)
Comment (Aimsucks) [reply README.md:62]: Resolved in [0a6ef22](https://github.com/ffxivcode/AutoDuty/pull/317/commits/0a6ef227ab04ddd4d6c742645894dd80caaa7406)

**Review by adambennett55 — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (Aimsucks) [reply README.md]: Resolved in [08d171f](https://github.com/ffxivcode/AutoDuty/pull/317/commits/08d171ffdf912c94222db68a7e03508701797170)

File: README.md (hunk 4 above)
Comment (Aimsucks) [reply README.md]: Resolved in [08d171f](https://github.com/ffxivcode/AutoDuty/pull/317/commits/08d171ffdf912c94222db68a7e03508701797170)

File: README.md (hunk 3 above)
Comment (Aimsucks) [reply README.md]: Resolved in [08d171f](https://github.com/ffxivcode/AutoDuty/pull/317/commits/08d171ffdf912c94222db68a7e03508701797170)

**Comment by Aimsucks** [comment]
I want to just add that the reason I don't link to channels in the Discord server is because if you are not a member of the server, it will take you to an error page instead of offering you the ability to join.

This was resolved by adding "this channel" links next to the "Puni.sh Discord server" links so people can refer to those if they're already members.

**Pull request merged**